source/install-by-copying-files/index.html.md.erb

```diff
@@ -0,0 +1,73 @@
+---
+title: Install by copying files
+weight: 10
+---
+
+# Install by copying files
+
+You can install GOV.UK Frontend by copying our CSS and JavaScript files into your project.
+
+<%= warning_text('We recommend <a href="/installing-with-npm/">installing with Node.js package manager (npm)</a> instead.
+') %>
+
+## What you cannot do
+
+If you install this way, you will not be able to:
+
+- change settings, for example you will not be able to override colours or set your own font
```

Comment:
Non-blocking because I'm not sure I've got a point... but is it clear that we're talking about doing things in Sass when we talk about 'not being able to override colours'? You can of course do what you like in your CSS file like overriding the colour of a Design System class (even if it's probably not very good practise to do that), maybe if you just wish to test something quickly.

Reply:
That's a good call - I'll tweak to "change Sass settings"